SR2022/2021-10-03-sr2022-competition-announcement.md

```diff
@@ -0,0 +1,30 @@
+---
+to: Potential team leaders
+subject: Announcing Student Robotics 2022
+---
+
+Hello there!
+
+We're pleased to announce sign-ups are now open for Student Robotics 2022!
+
+## Competition
+
+This year is the long-awaited return of the physical robotics competition! We're really excited to be going back to what we do best: An intense, exciting, and adrenaline-fuelled competition.
+
+In order to allow the core team of volunteers to focus on delivering a high quality programme while requiring far fewer resources than before, this year's competition will be slightly smaller than previous years. We expect to be able to increase the size of the competition again in the future.
+
+## Kickstart
+
+Kickstart will be streamed live on **13th November 2021**. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.
+
+If you already have a kit from a previous Student Robotics competition, please keep hold of it. If not, we will contact you soon to coordinate shipping a kit out to you.
+
+Visit [our website](TODO) for more information.
+
+## Sign up
+
+If you would like to compete in Student Robotics 2019, please fill in the form at the link below with the required information. All further emails about SR2019 will be sent there. The closing date will be *6th October*.
```

Comment:
Year fix:
```suggestion
If you would like to compete in Student Robotics 2022, please fill in the form at the link below with the required information. All further emails about SR2022 will be sent there. The closing date will be *6th October*.
```

Separately:
> All further emails about SR2022 will be sent there

This reads a bit oddly in context -- the inferred "there" is the signup form.
This also implies that we won't send any chasing emails to this list, which I suspect is limiting ourselves overly.

Suggest maybe saying something like this?:
> Details about the competition will be sent to the directly to the details you provide in the form.

I'd even be tempted to just omit the sentence; it seems relatively obvious to me that follow-on details are only going to be sent to people who sign up.

Reply:
It may seem obvious, but what I don't want is people thinking "I won't sign up yet, see how the details unfold" and then never hear about said details